src/components/AgreementSections/LinesList.js

```diff
@@ -5,6 +5,7 @@ import { FormattedMessage } from 'react-intl';
 import { get } from 'lodash';
 import { MultiColumnList } from '@folio/stripes/components';
 import { Spinner } from '@folio/stripes-erm-components';
+import { Tooltip } from '@folio/stripes-components';
```

Comment:
`Tooltip` should be imported via `@folio/stripes/components`. 

Anything that's re-exported by [`stripes`](https://github.com/folio-org/stripes) should never be imported directly, it should be pulled via Stripes itself. 

(I know this is kinda confusing when you see `stripes-erm-components` right above it, but `stripes-erm-components` isn't a part of Stripes [as you can see in Stripes' Github repo], but rather a package of ERM-related components for use in Stripes.)